default/empire_statistics.txt

```diff
@@ -1,3 +1,11 @@
+
+Statistic name = "MILITARY_STRENGTH_STAT" value =
+    Sum value = (LocalCandidate.Attack * (LocalCandidate.Structure + 2 * LocalCandidate.MaxShield))
```

Comment:
> The same result could probably have been scripted by using 2 Sum statistics added together instead of a single Sum with an add in the value.

If it was just an add in the value, then yes.  But it's not-- the extra handling was needed because there is also a multiply in there.